usr/src/uts/common/fs/zfs/lua/lstrlib.c

```diff
@@ -34,7 +34,6 @@
  */
 #define tolower(C)	(((C) >= 'A' && (C) <= 'Z') ? (C) - 'A' + 'a' : (C))
 #define toupper(C)      (((C) >= 'a' && (C) <= 'z') ? (C) - 'a' + 'A': (C))
-#define iscntrl(C)	((((C) >= 0) && ((C) <= 0x1f)) || ((C) == 0x7f))
```

Comment:
Indeed, and there's a https://www.illumos.org/issues/6306, which doesn't seem to be doable until the lint is gone.